modules/cnf-performing-end-to-end-tests-for-platform-verification.adoc

```diff
@@ -493,6 +493,36 @@ $ docker run -v $(pwd)/:/kubeconfig:Z -e KUBECONFIG=/kubeconfig/kubeconfig -e
 CLEAN_PERFORMANCE_PROFILE="false" registry.redhat.io/openshift-kni/cnf-tests /usr/bin/test-run.sh
 ----
 
+[id="cnf-performing-end-to-end-tests-running-in-single-node-cluster_{context}"]
+== Running in a single node cluster
+
+Running tests on a single node cluster causes the following limitations to be imposed:
+
+* Longer timeouts for certain tests, including SR-IOV and SCTP tests
+* Tests requiring master and worker nodes are skipped
+
+Longer timeouts concern SR-IOV and SCTP tests. Reconfiguration requiring node reboots cause a reboot of the entire environment, including the OpenShift control plane, and therefore takes longer to complete. All PTP tests requiring a master and worker node are skipped. No additional configuration is needed because the tests check for the number of nodes at startup and adjust test behavior accordingly.
+
+PTP tests can run in Discovery mode. The tests look for a PTP master configured outside of the cluster.
+
+For more information, see the Discovery mode section.
+// TODO update to xref
+
+To enable Discovery mode, the tests must be instructed by setting the `DISCOVERY_MODE` environment variable as follows:
+
+[source,terminal]
+----
+$ docker run -v $(pwd)/:/kubeconfig:Z -e KUBECONFIG=/kubeconfig/kubeconfig -e
+DISCOVERY_MODE=true registry.redhat.io/openshift-kni/cnf-tests /usr/bin/test-run.sh
+----
+
+[discrete]
+=== Required parameters
+
+* `ROLE_WORKER_CNF=master` - Required because master is the only machine pool to which the node will belong.
+* `XT_U32TEST_HAS_NON_CNF_WORKERS=false` - Required to instruct the xt_u32 negative test to skip because there are only nodes where the module is loaded.
```

Comment:
we also need the `SCTPTEST_HAS_NON_CNF_WORKERS=false` to instruct the sctp negative test to skip.
for the same reason of: `XT_U32TEST_HAS_NON_CNF_WORKERS=false`

Reply:
> we also need the `SCTPTEST_HAS_NON_CNF_WORKERS=false` to instruct the sctp negative test to skip.
> for the same reason of: `XT_U32TEST_HAS_NON_CNF_WORKERS=false`

@marzianor9 I have made this change. Please verify and let me know of any further changes.
Thank you.

Reply:
Great